xml/common_intro_feedback_i.xml

```diff
@@ -31,14 +31,17 @@
    <term>Bugs and Enhancement Requests</term>
    <listitem>
     <para>
-     For services and support options available for your product, refer to
-     <link xlink:href="http://www.suse.com/support/"/>.
+     Since Ceph for Windows is still in Beta, we would like to remind you that all
+     SUSE Beta Software are bound to our <link xlink:href="https://documentation.suse.com/beta/eula">SUSE Beta EULA</link>
+     Thus we are providing support via our Beta Program Channel.
     </para>
     <para>
-     To open a service request, you need a subscription at SUSE Customer
-     Center. Go to <link xlink:href="https://scc.suse.com/support/requests"/>,
-     log in, and click <guimenu>Create New</guimenu>.
+     To open a bug report, please follow the following <link xlink:href="https://suse.com/betaprogram/storage-beta/#bugzilla">link</link>.
```

Comment:
```suggestion
     To open a bug report, please follow the following instructions: <link xlink:href="https://suse.com/betaprogram/storage-beta/#bugzilla"/>.
```

Reply:
We intentionally do not use text in links, so that in print or non-hypertext formats, the reader can actually read the URL.

Reply:
I thought it was recommended to have Keyword-Rich Anchor Text to improve SEO (https://moz.com/learn/seo/anchor-text) ? What's your point of view on this?